src/main/scala/sbtecr/EcrPlugin.scala

```diff
@@ -7,6 +7,8 @@ import sbt._
 import scala.language.postfixOps
 
 object EcrPlugin extends AutoPlugin {
+  
+  lazy val version:SettingKey[String] = settingKey[String]("The ECR docker image version.")
```

Comment:
Thanks for the explanation - I've missed this conflict somehow. 

I think `version` scoped to `ecr` feels more natural indeed. In fact, I'm beginning to wonder whether we shouldn't move all AWS/ECR keys out of `autoImport`. It could become an issue when using different plugins for AWS-related tasks and none of these feel explicit enough for the "global" namespace (out of scope of this PR).